metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py

```diff
@@ -288,6 +292,24 @@ def get_workunits(self) -> Iterable[MetadataWorkUnit]:
                     self.report.report_workunit(wu)
                     yield wu
 
+                group_origin_mcp = MetadataChangeProposalWrapper(
+                    entityType="corpGroup",
+                    entityUrn=datahub_corp_group_snapshot.urn,
+                    changeType=ChangeTypeClass.UPSERT,
+                    aspectName="origin",
```

Comment:
super minor thing - can we use AZURE_AD here?

Reply:
It has been changed!